examples/styling/visuals/visible_annotation_with_interaction.py

```diff
@@ -1,12 +1,12 @@
 from bokeh.layouts import layout
-from bokeh.models import BoxAnnotation, Toggle
+from bokeh.models import BoxAnnotation, Node, Toggle
 from bokeh.plotting import figure, show
 
 p = figure(width=600, height=200, tools='')
 p.line([1, 2, 3], [1, 2, 1], line_color="#0072B2")
 pink_line = p.line([1, 2, 3], [2, 1, 2], line_color="#CC79A7")
 
-green_box = BoxAnnotation(left=1.5, right=2.5, fill_color='#009E73', fill_alpha=0.1)
+green_box = BoxAnnotation(left=1.5, right=2.5, top=Node.frame.top, bottom=Node.frame.bottom, fill_color='#009E73', fill_alpha=0.1)
```

Comment:
Is this for demonstration, or required? I don't think having to explicitly specify this is an improvement, or what users would normally want to have to do to have the box be "unbounded" in a direction.

Reply:
It's now required. Note that `BoxAnnotation` doesn't imply that it's bound to frame in any capacity (it's just a historical artifact), so unspecified positioning properties don't give a clue what the behavior will be (i.e. will it span, will it show at all?). It should be possible to attach it to anything like canvas or a panel or another box annotation. The syntax is indeed verbose, but at least the behavior is clear. `BoxAnnotation` wasn't ever a truly unbounded annotation, because it always draws lines on all sides and doesn't hit test at infinity, two things that `{H,V}Strip` glyphs do.